validation/validation_test.go

```diff
@@ -145,11 +145,13 @@ func (s *MySuite) TestValidateStep(c *C) {
 	valErr := specVal.validateStep(myStep)
 
 	c.Assert(valErr, Not(Equals), nil)
-	c.Assert(valErr.Error(), Equals, "foo.spec:3 Step implementation not found => 'my step'"+"\n"+
-		"Suggestion : "+"\n"+
-		"@Step(\"my step\")"+"\n"+
-		"public void implementation(){"+"\n\t"+
-		"// your code here...\n}")
+	c.Assert(valErr.Error(), Equals, "foo.spec:3 Step implementation not found => 'my step'")
+	c.Assert(valErr.(StepValidationError).Suggestion(), Equals, "One or more errors were due to : Step implementation not found => 'my step'\t\n"+
+		"Suggestion : \n"+
+		"@Step(\"my step\")\n"+
```

Comment:
@kashishm - fair enough. I think that the language runner should be able to set the suggestion. So `python` may choose to not indent, whereas other's might. From Gauge's point of view, it should print whatever the runner suggests.